src/stego.c

```diff
@@ -54,6 +66,7 @@ int main(int argc, char *argv[])
         printf("Input file: %s\n", input_file_name);
         printf("Porter file: %s\n", porter_file_name);
         printf("Output file: %s\n", output_file_name);
+        printf("Stego type: %s\n", stego_type->name);
```

Comment:
Replace with erasable logs

```suggestion
        LOG("Input file: %s\n", input_file_name);
        LOG("Porter file: %s\n", porter_file_name);
        LOG("Output file: %s\n", output_file_name);
        LOG("Stego type: %s\n", stego_type->name);
```